rskj-core/src/test/java/co/rsk/peg/BridgeSupportSvpTest.java

```diff
@@ -112,12 +116,123 @@ void setUp() {
         );
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("Validation failure tests")
+    class ValidationFailureTests {
+        @BeforeEach
+        void setUp() {
+            logs = new ArrayList<>();
+            BridgeEventLogger bridgeEventLogger = new BridgeEventLoggerImpl(
+                bridgeMainNetConstants,
+                allActivations,
+                logs
+            );
+
+            InMemoryStorage storageAccessor = new InMemoryStorage();
+            FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+            federationStorageProvider.setProposedFederation(proposedFederation);
+
+            federationSupport = new FederationSupportImpl(
+                bridgeMainNetConstants.getFederationConstants(),
+                federationStorageProvider,
+                rskExecutionBlock,
+                allActivations
+            );
+
```

Comment:
a real fed support impl is needed so when can check the proposed federation value was cleared

Reply:
You can just assert the method is being called and write unit tests for that method, as said above. There, you can check that the value is cleared.